src/com/dabomstew/pkrandom/gui/RandomizerGUI.form

```diff
@@ -1962,10 +2017,10 @@
                           </Group>
                           <EmptySpace type="unrelated" max="-2" attributes="0"/>
                           <Group type="103" groupAlignment="0" attributes="0">
-                              <Group type="102" attributes="0">
```

Comment:
this change results in unmatched tags. this is usually caught in compilation, but since it wasn't it's possibly causing bugs somewhere